ccdb/index.md

```diff
@@ -6,33 +6,46 @@ layout: ccdb
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/a/assets/201503_cfpb_Narrative-Scrubbing-Standard.pdf">
     <span class="icon-link_text">Scrubbing Standard</span>
-</a> and 
+</a> and
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/201509_cfpb_evaluation-of-narrative-scrubbing-standard-and-process.pdf">
     <span class="icon-link_text">Third Party Study</span>
 </a> for more details.
 
-We don’t verify all the facts alleged in these complaints, but we take steps to confirm a commercial relationship between the consumer and the company. We may remove complaints if they don’t meet all of the publication criteria as defined in our 
+We don’t verify all the facts alleged in these complaints, but we take steps to confirm a commercial relationship between the consumer and the company. We may remove complaints if they don’t meet all of the publication criteria as defined in our
 <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/201303_cfpb_Final-Policy-Statement-Disclosure-of-Consumer-Complaint-Data.pdf">
     <span class="icon-link_text">policy statement</span>
-</a>. Data is generally refreshed nightly. Company level information should be considered in context of company size and/or market share.
+</a>. Data is generally refreshed daily. Company level information should be considered in context of company size and/or market share.
 
-The Consumer Complaint Database contains complaints we've received about consumer financial products and services, including:
+The Consumer Complaint Database contains complaints we’ve received about consumer financial products and services. In April 2017 we updated the form consumers use to submit complaints. The changes include reorganizing how products, sub-products, issues, and sub-issues are grouped and making some plain language improvements.
 
-- Bank accounts or services
-- Consumer loans
-- Credit cards
-- Credit reporting
+The Consumer Complaint Database shows the consumer’s original product, sub-product, issue, and sub-issue selections consistent with the options available on the form at the time the consumer submitted the complaint.
+
+<ul class="list list__links">
+  <li class="list_item">
+    <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/documents/201704_cfpb_Summary_of_Product_and_Sub-product_Changes.pdf">
+      <span class="icon-link_text">Learn more about complaint form changes to products and sub-products</span>
+    </a>
+  </li>
+  <li class="list_item">
+    <a class="icon-link icon-link__pdf icon-link__no-wrap" href="http://files.consumerfinance.gov/f/documents/201704_cfpb_Consumer_Complaint_Form_Product_and_Issue_Options.pdf">
+      <span class="icon-link_text">View full list of complaint form products, sub-products, issues and sub-issues</span>
```

Comment:
Fixed in https://github.com/cfpb/api/pull/61/commits/154ca4af900c07fc58563ecfb942d62e4fa7439a